pkg/framework/spi/crypto/api.go

```diff
@@ -0,0 +1,42 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package crypto
+
+import "github.com/google/tink/go/keyset"
+
+// Provider interface for Crypto ctx
+type Provider interface {
+}
+
+// Creator method to create new pluggable Crypto service
+type Creator func(prov Provider) (Crypto, error)
+
+// Crypto is an interface defining a pluggable Crypto operations provider to Aries agents.
+// It's only responsible for pure crypto operations and nothing more. Envelope building and keys management are managed
```

Comment:
This line is superfluous.

Reply:
it's to highlight the limited scope of this interface, ie it's not responsible for getting keys or building jwes

Reply:
will remove

Reply:
done